modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE customers YIELD name, requestedStatus, currentStatus
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------+
+| name        | requestedStatus | currentStatus |
++-----------------------------------------------+
+| "customers" | "offline"       | "offline"     |
++-----------------------------------------------+
+----
+
+[NOTE]
+====
+Databases that are stopped with the `STOP` command are completely shut down and may be started again through the `START` command.
+In a cluster, as long as a database is in a shutdown state, it can not be considered available to other members of the cluster.
+It is not possible to do online backups against shutdown databases and they need to be taken into special consideration during disaster recovery, as they do not have a running Raft machine while shutdown.
+Dropped databases are completely removed and are not intended to be used again at all.
+====
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-start]]
+== Start databases
+
+Databases can be started using the command `START DATABASE`.
+
+.Query
+[source, cypher]
+----
+START DATABASE customers
+----
+
+[NOTE]
+====
+Both standard databases and composite databases can be stopped using this command.
+====
+
+The status of the started database can be seen using the command `SHOW DATABASE name`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE customers YIELD name, requestedStatus, currentStatus
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------+
+| name        | requestedStatus | currentStatus |
++-----------------------------------------------+
+| "customers" | "online"        | "online"      |
++-----------------------------------------------+
+----
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-delete]]
+== Delete databases
+
+Databases can be deleted by using the command `DROP DATABASE`.
+
+.Query
+[source, cypher]
+----
+DROP DATABASE customers
+----
+
+When a database has been deleted, it no longer shows up in the listing provided by the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES YIELD name
+----
+
+.Result
+[role="queryresult]
+----
++----------+
+| name     |
++----------+
+| "movies" |
+| "neo4j"  |
+| "system" |
++----------+
+----
+
+The `DROP DATABASE` command is optionally idempotent, with the default behavior to fail with an error if the database does not exist.
+Appending `IF EXISTS` to the command ensures that no error is returned and nothing happens should the database not exist.
+It will always return an error if there is an existing alias that targets the database.
+In that case, the alias needs to be dropped before dropping the database.
+
+.Query
+[source, cypher]
+----
+DROP DATABASE customers IF EXISTS
+----
+
+The `DROP DATABASE` command will remove a database entirely.
```

Comment:
it might be nice to mention something about it also dropping composite databases, similar to the note in start database